instructors/4-gis-slides.qmd

```diff
@@ -800,15 +732,15 @@ Join the community!
 :::
 :::
 
-## Call for helpers!
+## Thank you
 
-This workshop is offered on a yearly basis.
+-   the **workshop team**: Aleks, Ana, Claudiu, Clémentine, Daniele,
+    Javi, Jérôme, Kyri, Manuel, Paula, Selin + TU Library & BK Faculty
```

Comment:
Make sure everyone is included here according to the website: https://4turesearchdata-carpentries.github.io/2025-02-10-tudelft/
I will also make sure on the intro slides that everyone is there.